main_helper/omni_offline_client.py

```
@@ -180,45 +181,73 @@ async def stream_text(self, text: str) -> None:
         if self.on_input_transcript:
             await self.on_input_transcript(text.strip())
         
+        # Retry策略：重试2次，间隔1秒、2秒
```

Ruff (0.14.6)

Comment contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF003)